### PR TITLE
Feature/getprops limit push down

### LINKED ISSUE #3811: GetProps supports Limit push down.
**Is your feature request related to a problem? Please describe.**

Related to #2533 
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Shylock-Hg**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### Issue(s) number: 
Close #3811 
## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/graph/planner/plan/Query.h [hunk 1]
```diff
@@ -1555,7 +1555,7 @@ class AppendVertices final : public GetVertices {
                     nullptr,
                     false,
                     {},
-                    0,
+                    -1,  // means no limit
```
Comment (jackwener) [inline src/graph/planner/plan/Query.h:1558]: `IndexScan` use `9223372036854775807` indicate no limitation.

File: src/storage/query/GetPropProcessor.cpp [hunk 2]
```diff
@@ -22,6 +22,9 @@ void GetPropProcessor::process(const cpp2::GetPropRequest& req) {
 
 void GetPropProcessor::doProcess(const cpp2::GetPropRequest& req) {
   spaceId_ = req.get_space_id();
+  // negative number means no limit
```
Comment (jackwener) [inline src/storage/query/GetPropProcessor.cpp:25]: The first letter of comments should be capitalized

File: src/graph/planner/plan/Query.h (hunk 1 above)
Comment (Shylock-Hg) [reply src/graph/planner/plan/Query.h:1558]: Negative number is more easy to check is there limit requirement?

File: src/storage/query/GetPropProcessor.cpp (hunk 2 above)
Comment (Shylock-Hg) [reply src/storage/query/GetPropProcessor.cpp:25]: Done.

**Review by critical27 — APPROVED** [review]

**Comment by critical27** [comment]
LGTM

**Review by cangfengzhs — APPROVED** [review]
LGTM

**Review by jackwener — APPROVED** [review]
LGTM

**Pull request merged**